### DISCUSSION TITLE
[Linux] Use Pikafish with XBoard guide

### DISCUSSION THREAD

**Discussion by hungpt-a8e** [official-pikafish/Pikafish#43] [Show and tell]
### Install `uci2wb`
For Ubuntu distributions:
`sudo apt install uci2wb`
For other distros, clone and build uci2wb:
```
git clone 'http://hgm.nubati.net/git/uci2wb.git'
cd uci2wb
make uci2wb
make prefix=/usr install
```
### Configure Pikafish as Xboard engine
`Engine > Load New 1st Engine...`
![2023-07-15-12:49:50-screenshot](https://github.com/official-pikafish/Pikafish/assets/49029378/65eb5062-7958-425c-b02c-d0026a8cd42d)

> **IMPORTANT**
> - Engine Command is `uci2wb pikafish`
> - Make sure the first three boxes are _unchecked_

`Engine > Engine #1 Settings...`
![2023-07-15-13:00:16-screenshot](https://github.com/official-pikafish/Pikafish/assets/49029378/c08aad04-1cf0-4c03-8537-d12dc4f4572e)

> **IMPORTANT**
> Make sure `EvalFile` field is configured and pointed to the correct Pikafish NNUE file. If you don't configure this field or the path to the file is wrong then the engine will die.

**Comment by PikaCat-OuO** [comment]
Thanks for your great guide, I have opened access to the wiki. If you would like to make it into the wiki you can put it on the wiki page.😊

**Reply by hungpt-a8e** [reply]
I've added the guide to the wiki's general guides.
